front-end/Image-recognition/src/app/recognition.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+//  mark the RecognitionService class as a service that can be injected into other components
+@Injectable({
+  providedIn: 'root'
+})
+export class RecognitionService {
+  // Inject the HttpClient service in the constructor
+  constructor(private http: HttpClient) { }
+
+  /**
+   * Sends a POST request to a recognition API to recognize a photo.
+   * @param photo The photo to recognize.
+   * @returns An observable of the response from the recognition API.
+   */
+  public RecognizePhoto(photo: any): any {
+    // Define the URL and data for the POST request
+    const apiUrl = 'http://localhost:4200/recognition';
+    const data = { photo: photo };
```

Comment:
I forget to mention the structure of the body we send to the back end 
please update the JSON file to match the back end  
{
content: photo_content
}

Reply:
DONE